teamA/initial-settings-express/heeyeon/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER SHOULD BE HERE!
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = ip host number
+TYPEORM_USERNAME = mysql의 계정(username)
+TYPEORM_PASSWORD = myPassword
+TYPEORM_DATABASE = DB명
+TYPEORM_PORT = typeORM의 포트번호
+TYPEORM_LOGGING = typeORM과 네트워크 통신시 실제로 어떻게 에러가 났고, 통신시 기타 참고사항이 있을지 등 로깅과 관련된 기록들을 한 곳에 모아두는 행위를 할 것인지 말 것인지 여부 기재
```

환경변수 샘플 파일은 해당 프로젝트를 같이 진행하는 타 개발자와 공유하는 데 그 목적이 있습니다.

다음과 같이 자세한 설명도 좋지만, 가독성을 위해 간단명료하게 적는 것도 하나의 방법이 될 수 있습니다~!

추가로, dbmate 연결 목적의 DATABASE_URL을 추가해주시고 로컬 환경에서 어떠한 rdbms를 쓰는지 그 힌트가 될 수 있는 단어와, 포트 번호를 지우고 중의적인 단어를 대신 사용해주세요.

```suggestion
DATABASE_URL="rdbmsType:username:password@host:portNumber/databaseName"

PORT = PORT_NUMBER

TYPEORM_CONNECTION = mysql
TYPEORM_HOST = HOST
TYPEORM_USERNAME = USERNAEM
TYPEORM_PASSWORD = PASSWORD
TYPEORM_DATABASE = DB_NAME
TYPEORM_PORT = DB_PORTNUMBER
TYPEORM_LOGGING = BOOLEAN
```